code/utils.py

```diff
@@ -115,7 +117,6 @@ def get_s3_input_information(
                 )
                 continue
             
-            s3_paths.append(location)
```

Comment:
I put this here intentionally - would say that if an extension is specified and not found in an asset, that asset should be skipped.